src/DynamoPackages/PackageLoader.cs

```diff
@@ -391,13 +394,17 @@ public void LoadCustomNodesAndPackages(IEnumerable<string> newPaths, IPreference
 
                 customNodeManager.AddUninitializedCustomNodesInPath(dir, false, false);
             }
-
-            foreach (var path in newPaths)
+            var paths = packageDirectoriesToScan ?? newPaths;
```

Comment:
I think `paths` should be used at line https://github.com/DynamoDS/Dynamo/pull/12444/files#diff-7ae21a0e381958737cc115ada84cbd9f96efe3315928cbc44b367967923cf55bR426